StantUiIosLibrary/Classes/Tables/GroupSelectionTableViewWithCollapsedSearchBar/GroupSelectionTableViewWithCollapsedSearchBar.swift

```diff
@@ -0,0 +1,148 @@
+        guard let searchView = searchView else { return }
+        self.addSubview(searchView)
+        searchView.configureViewWith(delegate: self,
+                                     image: searchBarIcon,
+                                     placeholderText: searchBarPlaceholder)
+    }
+    
+    
+    fileprivate func configureTableView() {
+        tableView = GroupSelectionTableView(frame: CGRect(x: 0,
+                                                          y: DefaultSearchBar.searchViewHeight,
+                                                          width: self.frame.width,
+                                                          height: self.frame.height),
+                                            style: .grouped)
+        
+        tableView?.register(ConstructionCard.self, forCellReuseIdentifier: ConstructionCard.IDENTIFIER)
+        tableView?.configureTableViewWith(itemsList: itemsList,
+                                          animationDelegate: self,
+                                          selectCellDelegate: tableViewDelegate)
+        tableView?.backgroundColor = UIColor.clear
+    
+        if let tableView = tableView { self.addSubview(tableView) }
+    }
+    
+    fileprivate func anchorSearchAndTableView() {
+        guard let tableView = tableView, let searchView = searchView else { return }
+        
+        searchView.anchor(top: self.topAnchor,
+                          leading: self.leadingAnchor,
+                          bottom: self.bottomAnchor,
+                          trailing: self.trailingAnchor,
+                          padding: UIEdgeInsets(top: 0,
+                                                left: 6,
+                                                bottom: self.frame.height - DefaultSearchBar.searchViewHeight,
+                                                right: 6))
+        tableView.anchor(top: searchView.topAnchor,
+                         leading: self.leadingAnchor,
+                         bottom: self.bottomAnchor,
+                         trailing: self.trailingAnchor,
+                         padding: UIEdgeInsets(top: DefaultSearchBar.searchViewHeight, left: 0, bottom: 0, right: 0))
+    }
+    
+    fileprivate func configureEmptyMessageLabel() {
+        emptyMessageLabel?.removeFromSuperview()
+        emptyMessageLabel = UILabel(frame: CGRect(x: 0,
+                                                  y: DefaultSearchBar.searchViewHeight,
+                                                  width: self.frame.width,
+                                                  height: self.frame.height))
+        
+        guard let emptyMessageLabel = emptyMessageLabel else { return }
+        emptyMessageLabel.configure(text: emptyMessage,
+                                    alignment: .left,
+                                    size: 16,
+                                    weight: .regular,
+                                    color: UIColor.darkStant)
```

Comment:
If i'm not wrong, you could also call the color as a property just as the same way that you did above with ```.regular```. (just a suggestion, if it works :))

Reply:
True story :) Changed.